tests/examples/pacer/nef/s3/pamb_4.json

```diff
@@ -0,0 +1,40 @@
+{
+  "appellate": false,
+  "contains_attachments": false,
+  "court_id": "pamb",
+  "dockets": [
+    {
+      "case_name": "Charlene R. House",
+      "date_filed": null,
+      "docket_entries": [
+        {
+          "date_filed": "2024-04-19",
+          "description": "BNC Certificate of Notice (341 Meeting Notice (Chapter 13)) (RE: related document(s)[7]). Notice Date 04/19/2024. (Admin.)",
+          "document_number": "8",
+          "document_url": "https://ecf.pamb.uscourts.gov/doc1/154018316665?pdf_header=&magic_num=88545762&de_seq_num=31&caseid=279888",
+          "pacer_case_id": "279888",
+          "pacer_doc_id": "154018316665",
+          "pacer_magic_num": "88545762",
+          "pacer_seq_no": "31",
+          "short_description": "BNC Certificate of Notice (341 Meeting Notice (Chapter 13))"
```

Comment:
We keep the "Chapter ..." string

Email subject:
`Subject:Ch-13 1:24-bk-00941-HWV -Charlene R. House BNC Certificate of Notice (341 Meeting Notice (Chapter 13))`

Document history report:
![image](https://github.com/user-attachments/assets/472a2d93-8b63-441d-a132-79a0ef77214e)